src/libs/runtime-utils.js

```diff
@@ -161,18 +175,19 @@ export const runtimeConfigCost = config => {
 }
 
 // Per GB following https://cloud.google.com/compute/pricing
-const getPersistentDiskPriceForRegionMonthly = computeRegion => {
-  return _.flow(_.find({ name: _.toUpper(computeRegion) }), _.get(['monthlyDiskPrice']))(regionToPrices)
+const getPersistentDiskPriceForRegionMonthly = (computeRegion, diskType) => {
+  const selectedRegionToPricesValue = pdTypes[_.replace('pd-', '', diskType)]?.regionToPricesName || pdTypes.standard.regionToPricesName
```

Comment:
Can we eliminate the need for the string manipulation on the line above, by having the input parameter `diskType` to have value `pdTypes.standard` or `pdTypes.ssd` so we can directly get `regionToPricesName` on it?